ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -45,6 +45,15 @@ const (
 	JobDeploymentStatusWaiting      JobDeploymentStatus = "Waiting"
 )
 
+var AllJobStatuses = []JobStatus{
```

Comment:
can we move it below L15 - L22, and add some comments above L15 to ensure that we will update `AllJobStatuses` if we add a new `JobStatus`?

Reply:
updated, unfornately `JobStatus` is not an integer value, so it's hard to add a compile-time check to make sure array / string number match.